website/docs/r/app_service.html.markdown

```diff
@@ -135,6 +135,8 @@ A `logs` block supports the following:
 
 * `application_logs` - (Optional) An `application_logs` block as defined below.
 
+* `http_logs` - (Optional) An `https_logs` block as defined below.
```

Comment:
```suggestion
* `http_logs` - (Optional) An `http_logs` block as defined below.
```

also - can both of these be set?

Reply:
Yes, they can both be set but I'll get a test to confirm it!